tests/fetchMode/data/modes.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace DefaultFetchModeTest;
+
+use PDO;
+use function PHPStan\Testing\assertType;
+
+class Foo
+{
+    public function assocModeQuery(PDO $pdo)
+    {
+        // default fetch-type is globally changed to assoc for this test-suite
+
+        $stmt = $pdo->query('SELECT email, adaid, gesperrt, freigabe1u1 FROM ada');
```

Comment:
there is another test-case when passing the `fetch-type` as a 2nd arg to `query()`